src/Valit/Rules/ValitRules.cs

```diff
@@ -2,93 +2,114 @@
 using System.Collections.Generic;
 using System.Linq;
 using Valit.MessageProvider;
+using System.Linq.Expressions;
+using Valit.Expressions;
 using Valit.Strategies;
 
 namespace Valit
 {
-    public class ValitRules<TObject> : 
-        IValitRules<TObject>,
-        IValitRulesMessageProvider<TObject>, 
-        IValitRulesStrategyPicker<TObject> 
-        where TObject : class
+    public static class ValitRules
     {
-        private TObject _object;       
-        private readonly List<IValitRule<TObject>> _rules;
+        public static IValitRulesStrategyPicker<TTarget> CreateFor<TTarget>(TTarget @object) where TTarget : class
```

Comment:
Ideally, we would like to create stateless validator, which would be build once and run on many models. That's why we changed Create<TModel>(TModel model) to just Create()

Reply:
That is true... but in the meantime... while we have this .For...